### DISCUSSION TITLE
Query vs Command Endpoints

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#552] [General]
Dear Developers,

the [HTTP endpoint for queries](https://docs.arcadedb.com/#HTTP-ExecuteQuery) has two variants:
```
/api/v1/query/{database}
```
and
```
/api/v1/query/{database}/{language}/{command}
```
while the [HTTP endpoint for commands](https://docs.arcadedb.com/#HTTP-Command) has only one variant:
```
/api/v1/command/{database}
```
Is this "final" and there is some reasoning, technical necessary or a best-practices behind this, or will this be changed in the future to either query and command both featuring both variants or both variants featuring only one variant?

**Comment by lvca** [comment]
The HTTP GET /query actually accepts parameters only on the request URL, not as payload. Just updated the docs.
